### ISSUE TITLE
Turtle appears not on the trailing line on Chrome & Edge browsers

### ISSUE THREAD

**Issue by nonkit** [sb/smallbasic-editor#107]
Turtle appears not on the trailing line on Chrome browser.

This is the sample code:

```
GraphicsWindow.DrawImage("https://aka.ms/smallbasic-maze1", 0, 0)
Turtle.Show()
Turtle.X = 50
Turtle.Y = 50
Turtle.Speed = 9
Turtle.TurnRight()
Turtle.Move(200)
Turtle.TurnRight()
Turtle.Move(200)
Turtle.TurnRight()
Turtle.Move(200)
Turtle.TurnLeft()
Turtle.Move(100)
```

Result with SBO 1.0:

![Turtle Chrome](https://user-images.githubusercontent.com/9944452/66297397-90dfe000-e92a-11e9-8b1f-914608e1519b.png)

Result with SBD 1.2:

![Turtle SBD 1 2](https://user-images.githubusercontent.com/9944452/66297423-9dfccf00-e92a-11e9-9b21-4a470c679117.png)

**Comment by Ed-Price** [comment]
This is also an issue now with the new Edge browser, which is powered by the same technology as Chrome.

**Comment by Ed-Price** [comment]
I just tested on Edge, and it's no longer an issue. I used Nonki's sample code at the top.

Note that if you have an Edge update, then installing/updating Edge will get this bug in the fixed state.

**Comment by Ed-Price** [comment]
I just tested on Chrome. Casey did as well. It is fixed.

**Comment by Ed-Price** [comment]
Verified fixed by multiple users on Edge (latest update required) and Chrome. Sending to Nonki to verify.

**Comment by nonkit** [comment]
I confirmed #107 fixed both on Edge 84.0.522.52 (Official build) (64-bit) and Chrome 84.0.4147.105 (Official Build) (64-bit).  Thanks!
